www/index.html

```diff
@@ -1,56 +1,85 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Hello wasm-nu</title>
-    <link rel="stylesheet" href="https://fonts.xz.style/serve/inter.css" />
-    <link
-      rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/@exampledev/new.css@1.1.2/new.min.css"
-    />
-
-    <style>
-      #examples {
-        display: flex;
-        column-gap: 1rem;
-      }
-      .output-error {
-        font-family: monospace;
-      }
-      .output-error--inline {
-        color: red;
-      }
-      td {
-        min-width: 6rem;
+
+<head>
```

Comment:
What code formatter are you using? I'm using prettier, which has different defaults than this.

Reply:
I use vscode, so I think it's whatever formatter it uses by default. Should I change it?

Reply:
I suggest we use https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode to get consistent output. We could add prettier as a CI tool for linting later.